### PR TITLE
feat: implement Bluetooth Emulation BiDi mapping

### PR DESCRIPTION
**Pull request by alexnj**
Implements the Bluetooth Emulation spec from WebBluetoothCG/web-bluetooth#630. Needs to wait until spec is landed before this can be merged.

### CODE REVIEW

File: src/protocol/chromium-bidi.ts [hunk 1]
```diff
@@ -110,6 +110,9 @@ export type Command = (
       // not re-define it. Therefore, it's not part of generated types.
       id: WebDriverBidi.JsUint;
     } & WebDriverBidiBluetooth.Bluetooth.HandleRequestDevicePrompt)
+  | ({
```
Comment (alexnj) [inline src/protocol/chromium-bidi.ts:113]: TODO: resolve via types from CDDL.

File: src/protocol/generated/webdriver-bidi-bluetooth.ts [hunk 2]
```diff
@@ -32,13 +32,26 @@ export namespace Bluetooth {
...
   export type RequestDevicePrompt = string;
+  export type AdapterState = 'powered-on' | 'powered-off' | 'absent';
 }
 export namespace Bluetooth {
   export type HandleRequestDevicePrompt = {
     method: 'bluetooth.handleRequestDevicePrompt';
     params: Bluetooth.HandleRequestDevicePromptParameters;
   };
 }
+export namespace Bluetooth {
```
Comment (alexnj) [inline src/protocol/generated/webdriver-bidi-bluetooth.ts:43]: TODO: resolve via types from CDDL.

**Review by OrKoN — APPROVED** [review]

**Review by sadym-chromium — APPROVED** [review]

**Comment by OrKoN** [comment]
@alexnj thanks for updating the types! I think they have been rolled now so this PR can be rebased and landed.

File: src/bidiMapper/CommandProcessor.ts [hunk 3]
```diff
@@ -148,6 +148,16 @@ export class CommandProcessor extends EventEmitter<CommandProcessorEventsMap> {
         return await this.#bluetoothProcessor.handleRequestDevicePrompt(
           this.#parser.parseHandleRequestDevicePromptParams(command.params)
         );
+      case 'bluetooth.simulateAdapter':
+        return await this.#bluetoothProcessor.simulateAdapter(command.params);
```
Comment (sadym-chromium) [inline src/bidiMapper/CommandProcessor.ts:152]: FYI: the params should be parsed

**Pull request merged**